cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConstant.java

```diff
@@ -47,8 +48,9 @@ private ClusterConstant() {
   }
 
   /**
-   * a failed election will restart in 2s~5s, this should be at least as long as a heartbeat
-   * interval, or a stale node may frequently issue elections and thus makes the leader step down
+   * a failed election will restart in [2*heartbeatInterval, 2*heartbeatInterval + 50ms). If this
```

Comment:
Acutally, I think the lower bound should be 0. 
With current logic, all candidates will wait this extra time for `2*heartbeatInterval`, which does not help to avoid live locks. It also does not help that stale nodes initiate elections first. 
What do you think?

Reply:
Yes, it's useless to set a minimum waiting time before starting election. After discussed with @jixuan1989 , we think that a constant maximum waiting time is not a good practice in real environment, but adding another configure parameter makes the cluster config more complicated. So we just set it as `1*heartbeatInterval`, which is lack of experiment now. The 50ms is just a protection if the heartbeatInterval is very small.